core/src/navigation/TopNav.html

```diff
@@ -581,6 +582,10 @@
     });
   }
 
+  export function closeSearchResult() {
+    dispatch('closeSearchResult');
+  }
+
```

Comment:
Just curious, this `closeSearchResult` function is same as the one at `GlobalSearch.html`. Is there any possibility to extract it out? and import it separately.

Reply:
Good catch! It's a left over, I remove it :)